geowebcache/wms/src/test/java/org/geowebcache/service/wms/WMSTileFuserTest.java

```diff
@@ -44,14 +47,17 @@
 import org.geowebcache.storage.StorageException;
 import org.geowebcache.storage.TileObject;
 import org.geowebcache.storage.blobstore.file.FileBlobStore;
+import org.mockito.Mockito;
 import org.springframework.context.support.ClassPathXmlApplicationContext;
 import org.springframework.mock.web.MockHttpServletRequest;
 import org.springframework.mock.web.MockHttpServletResponse;
 
 public class WMSTileFuserTest extends TestCase {
     GridSetBroker gridSetBroker = new GridSetBroker(false, false);
+    SecurityDispatcher secDisp = mock(SecurityDispatcher.class);
     
     public void testTileFuserResolution() throws Exception {
+        
```

Comment:
If you are adding the security dispatcher here, maybe also add one test for a security violation?

Reply:
Done